bot.js

```diff
@@ -7,61 +7,50 @@ const bot = new Discord.Client();
 const token = process.env.TOKEN;
 const prefix = process.env.PREFIX;
 
-bot.on('ready', () => {
-	console.log(`Logged in as ${bot.user.username}!`);
-});
-
+bot.on('ready', () => console.log(`Logged in as ${bot.user.username}!`));
```

Comment:
Get this back the way it was before :rage:

Reply:
Why? It's just one line.

Reply:
it's prettier, though